apps/rehash.c

```diff
@@ -53,7 +53,7 @@
 
 # if defined(OPENSSL_SYS_VXWORKS)
 /* 
-    VxWorks has no symbolic links 
+*   VxWorks has no symbolic links 
 */
```

Comment:
One more space in 56 and 57, to align the `*`

https://github.com/openssl/openssl/blob/e9e0dadf5c4f57c7ddb8dfd6310bbc77901d5c80/apps/rehash.c#L55-L57